src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -0,0 +1,267 @@
+            await vscode.window.showTextDocument(vscode.Uri.file(file.absPath));
+        });
+    }
+
+    public setQueuedPipelineUrl(provisioningConfiguration: ProvisioningConfiguration, inputs: WizardInputs){
+      const commitId = (provisioningConfiguration.result.pipelineConfiguration as CompletePipelineConfiguration).commitId;
+      this.queuedPipelineUrl = `https://github.com/${inputs.sourceRepository.repositoryId}/commit/${commitId}/checks`;
+    }
+
+    public async  getFileToCommit(draftPipelineConfiguration: DraftPipelineConfiguration): Promise<void> {
+        let destination: string;
+        for (const file of draftPipelineConfiguration.files ) {
+            destination = await this.getPathToFile(Path.basename(file.path), Path.dirname(file.path));
+            const decodedData = new Buffer(file.content, 'base64').toString('utf-8');
+            this.filesToCommit.push({absPath: destination, content: decodedData, path: file.path} as DraftFile);
+        }
+    }
+
+    public async getPathToFile( fileName: string, directory: string) {
+        const dirList = directory.split("/"); // Hardcoded as provisioning service is running on linux and we cannot use Path.sep as it is machine dependent
+        let directoryPath: string = "";
+        directoryPath = await this.localGitRepoHelper.getGitRootDirectory();
+        dirList.forEach((dir) => {
+            try {
+                directoryPath = Path.join(directoryPath, dir);
+                // tslint:disable-next-line:non-literal-fs-path
+                if (!fs.existsSync(directoryPath)) {
+                    // tslint:disable-next-line:non-literal-fs-path
+                    fs.mkdirSync(directoryPath);
+                }
+            }
+            catch (error) {
+                throw error;
+            }
+        });
+        telemetryHelper.setTelemetry(TelemetryKeys.WorkflowFileName, fileName);
+        return Path.join(directoryPath, fileName);
+    }
+
+    public async CreatePreRequisiteParams(wizardInputs: WizardInputs): Promise<void>{
+        // Create SPN and ACRResource group for reuseACR flow set to false
+        const inputDescriptor = this.getInputDescriptor(wizardInputs, "azureAuth");
+        if (inputDescriptor != undefined){
+            const createResourceGroup = InputControl.getInputDescriptorProperty(inputDescriptor, "createResourceGroup", wizardInputs.pipelineConfiguration.params);
+            if ( createResourceGroup.length > 0 && createResourceGroup[0] != "" ){
+                await vscode.window.withProgress(
+                    { location: vscode.ProgressLocation.Notification, title: Messages.CreatingACRResourceGroup },
+                    async () => {
+                        try {
+                           return await new ArmRestClient(wizardInputs.azureSession).createResourceGroup(wizardInputs.subscriptionId, createResourceGroup[0], wizardInputs.pipelineConfiguration.params["containerRegistryLocation"]);
```

Comment:
Okay we need location as well, and I think it is wrong to take dependency on containerRegistryLocation as we might be creating RG for some other purpose in future. Should we take an array of  <rg name, location> instead of rg name?

Reply:
/cc @anuragc617

Reply:
Added changes to take location from template. Similar changes are at template service as well.